src/Image.zig

```diff
@@ -211,18 +179,11 @@ pub fn imageByteSize(self: Self) usize {
 
 /// Is this image is an animation?
 pub fn isAnimation(self: Self) bool {
-    return switch (self.data) {
-        .animation => true,
-        else => false,
-    };
+    return self.animation != null;
```

Comment:
Should this also include `and self.animation.frames.len > 0` since you check that everywhere as well. Plus you could replace all those double ifs with a single `if (self.isAnimation())`. On the other hand if it was not nullable you could just check the len :)